docker/report_collector.Dockerfile

```diff
@@ -0,0 +1,24 @@
+# syntax=docker/dockerfile:1
+ARG SOURCES_DIR=/usr/src/ipa
+FROM rust:bookworm AS builder
+ARG SOURCES_DIR
+
+# Prepare report collector binaries
+WORKDIR "$SOURCES_DIR"
+COPY . .
+RUN set -eux; \
```

Comment:
report collector only needs two features "cli" and "test-fixture"

Reply:
I checked and it needs all those features